### ISSUE TITLE
What about desktop support?

### ISSUE THREAD

**Issue by AntonLT** [GiancarloCode/form_bloc#220]

**Comment by WahdanZ** [comment]
by default the package support all platforms, due face any problems with any platform

**Comment by WahdanZ** [comment]
by default the package support all platforms, due face any problems with any platform

**Comment by AntonLT** [comment]
Yeah, the used package https://pub.dev/packages/flutter_keyboard_visibility has no desktop support and causes warnings and sometimes even crushes on the desktop.
I personally always have crushes when tabbing on checkbox group widgets.
